src/structures/MessageSelectMenu.js

```diff
@@ -43,6 +46,16 @@ class MessageSelectMenu extends BaseMessageComponent {
   constructor(data = {}) {
     super({ type: 'SELECT_MENU' });
 
+    if (!emittedDeprecationNotice) {
+      process.emitWarning(
+        `MessageSelectMenu is deprecated and will be removed in a future release,
+         please use SelectMenuComponent instead`,
```

Comment:
Pretty sure this would have a huge gap in the middle, so please update this one to match the others